src/vaults/vaults.utils.ts

```diff
@@ -463,6 +464,29 @@ function constructGraphVaultData(
   });
 }
 
+export async function estimateHarvestEventApr(
+  chain: Chain,
+  vault: VaultDefinition,
+  start: number,
+  end: number,
+  amount: VaultPerformanceEvent['amount'],
+  balance: number,
+): Promise<number> {
+  const duration = end - start;
+
+  const depositToken = await getFullToken(chain, vault.depositToken);
+
+  const fmtBalance = formatBalance(balance, depositToken.decimals);
+
+  const measuredBalance = fmtBalance / duration;
```

Comment:
i think we can drop this part since we want to use the full balance during the interval